pkg/operator/ceph/cluster/cluster.go

```diff
@@ -525,6 +525,17 @@ func (c *cluster) postMonStartupActions() error {
 		return errors.Wrap(err, "failed to create cluster rbd bootstrap peer token")
 	}
 
+	// Delete bootstrap keys
+	// TODO: add additional note on why these keys are deleted.
```

Comment:
Suggest: `client.bootstrap-*` keys are created automatically by Ceph and are unused. In the future, Ceph team has indicated that they will remove these. It is safe for Rook to remove them if they exist in 'legacy' clusters. `bootstrap-osd` is used by Rook and is not removed here.